### ISSUE TITLE
Put OMGX branding in the README.md

### ISSUE THREAD

**Issue by mmontour1306** [omgnetwork/optimism#250]
I noticed that when I go to the https://github.com/omgnetwork/optimism page, the default view shows a README.md which includes the upstream Optimism branding. It would be good if we edited our copy of this file to clarify that we're working on our own distinct fork of the project in our own monorepo. This will be particularly important when we're trying to attract new developers from the community.

Relevant sections of README.md:

<div align="center">
  <a href="https://community.optimism.io"><img alt="Optimism" src="https://user-images.githubusercontent.com/14298799/122151157-0b197500-ce2d-11eb-89d8-6240e3ebe130.png" width=280></a>
  <br />
  <h1> The Optimism Monorepo</h1>
</div>
[...] 
## TL;DR

This is the primary place where [Optimism](https://optimism.io) works on stuff related to [Optimistic Ethereum](https://research.paradigm.xyz/optimism).
[...]
